src/test/java/plugins/JobDslPluginTest.java

```diff
@@ -817,6 +822,615 @@ public void should_alert_if_generated_job_changed() {
+                "}";
+        openNewlyCreatedListView(jobDslScript, LIST_VIEW_NAME);
+        WebElement description = find(By.xpath("//div[@id='description']/div"));
+        Assert.assertThat(description.getText(), containsString(descriptionText));
+    }
+
+    /**
+     * Test if all native columns of a ListView are created and shown correctly.
+     */
+    @Test
+    public void are_columns_created() {
+        Job job1 = createJobAndBuild();
+        String jobDslScript = "listView('"+ LIST_VIEW_NAME +"') {\n" +
+                "  columns {\n" +
+                "    status()\n" +
+                "    weather()\n" +
+                "    name()\n" +
+                "    lastSuccess()\n" +
+                "    lastFailure()\n" +
+                "    lastDuration()\n" +
+                "    buildButton()\n" +
+                "  }\n" +
+                "  jobs{\n" +
+                "    name('"+job1.name+"')\n" +
+                "  }\n" +
+                "}";
+        openNewlyCreatedListView(jobDslScript, LIST_VIEW_NAME);
+        Assert.assertThat(driver, hasElement(By.xpath("//th[contains(@tooltip, 'Status of the last build')]")));
+        Assert.assertThat(driver, hasElement(By.xpath("//th[contains(@tooltip, 'Weather report showing aggregated status of recent builds')]")));
+        Assert.assertThat(driver, hasElement(By.xpath("//th/a[text() = 'Name']")));
+        Assert.assertThat(driver, hasElement(By.xpath("//th/a[text() = 'Last Success']")));
+        Assert.assertThat(driver, hasElement(By.xpath("//th/a[text() = 'Last Failure']")));
+        Assert.assertThat(driver, hasElement(By.xpath("//th/a[text() = 'Last Duration']")));
+        Assert.assertThat(driver, hasElement(By.xpath("//td/a/img[contains(@src, 'clock.png')]")));
+    }
+
+    /**
+     * Test if the created jobs are correctly added to the ListView and if the status filter works correctly.
+     * In this case only enabled jobs shall be shown.
+     */
+    @Test
+    public void status_filter_only_shows_enabled_jobs() {
+        List<Job> jobs = createAmountOfJobs(3, false);
+        String jobDisabled = "exampled-disabled";
+        String jobDslScript = "job('"+jobDisabled+"') {\n" +
+                "  disabled()\n" +
+                "}\n" +
+                "listView('"+ LIST_VIEW_NAME +"') {\n" +
+                "  statusFilter(StatusFilter.ENABLED)\n" +
+                "  columns {\n" +
+                "    name()\n" +
+                "  }\n" +
+                "  jobs {\n" +
+                "    names('"+jobs.get(0).name+"', '"+jobs.get(1).name+"', '"+jobs.get(2).name+"', '"+jobDisabled+"')\n" +
+                "  }\n" +
+                "}";
+        openNewlyCreatedListView(jobDslScript, LIST_VIEW_NAME);
+        Assert.assertThat(driver, not(hasElement(By.xpath("//tbody/tr[contains(@id, '"+jobDisabled+"')]"))));
+        Assert.assertThat(driver, hasElement(By.xpath("//tbody/tr[contains(@id, '"+jobs.get(0).name+"')]")));
```

Comment:
if the right part of this line could not be placed into a method in the View class then there should be at least a helper method in this test case that avoids the duplications.